docs/avatar.md

```diff
@@ -0,0 +1,43 @@
+# Avatar
+
+## Mixin
+
+`user_management.models.mixins.AvatarMixin` adds an avatar field. The 
+serializers require `django-imagekit`.
+
+## Installation
+
+Install with avatar functionality
+
+    pip install django-user-management[avatar]
+
+Add the urls to your `ROOT_URLCONF`
+
+    urlpatterns = patterns(''
+        ...
+        url('', include('user_management.api.avatar.urls.avatar')),
+        ...
+    )
+
+## View
+
+`user_management.api.avatar.views.ProfileAvatar` provides an endpoint to retrieve 
+and update the logged in user's avatar.
+
+`user_management.api.avatar.views.UserAvatar` provides an endpoint to retrieve 
+and update other user's avatar. Only admin user can update other user's data.
+
+Both avatar views provides an endpoint to retrieve a thumbnail of the
+authenticated user's avatar.
+
+    Thumbnail options can be specified as get parameters. Options are:
```

Comment:
I think this section (lines 33-43 at the moment) needs to be de-indented, because it renders oddly:

![screenshot from 2015-03-17 11 06 15](https://cloud.githubusercontent.com/assets/6973481/6685898/eedb3350-cc95-11e4-88df-94b7ec09ecbb.png)